.github/workflows/all-modules-add-and-buildable.yaml

```diff
@@ -5,10 +5,9 @@ name: Validate all modules in index can be added and built
 # Controls when the workflow will run
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
```

Comment:
Maybe remove/change this comment, as it no longer applies for push or pull ?

Reply:
ack